pytests/src/lib.rs

```diff
@@ -18,43 +15,39 @@ pub mod sequence;
 pub mod subclassing;
 
 #[pymodule]
-fn pyo3_pytests(py: Python<'_>, m: &PyModule) -> PyResult<()> {
-    m.add_wrapped(wrap_pymodule!(awaitable::awaitable))?;
-    #[cfg(not(Py_LIMITED_API))]
-    m.add_wrapped(wrap_pymodule!(buf_and_str::buf_and_str))?;
-    m.add_wrapped(wrap_pymodule!(comparisons::comparisons))?;
+mod pyo3_pytests {
+    use super::*;
+    use pyo3::types::PyDict;
+    #[pyo3]
```

Comment:
Going off the "export" suggestion of yours, how about `#[pymodule_export]` for now? That has a nice consistency with `#[pymodule_init]`. We can always improve these names again later.

Reply:
Perfect! Done!